runtime/strategies/search-tokens-to-particles.js

```diff
@@ -17,9 +15,9 @@ module.exports = class SearchTokensToParticles extends Strategy {
     // TODO: Recipes. Views?
     this._byToken = {};
     for (let particle of arc.context.particles) {
-      let name = particle.name.toLowerCase();
-      this._byToken[name] = this._byToken[name] || [];
-      this._byToken[name].push(particle);
+      let verb = particle.primaryVerb;
```

Comment:
Should we support both particle and name here?

Reply:
right, updated.
i also had to update the strategy itself to properly support the case with multiple particles corresponding to the same verb. and added a test case.